src/3rdparty/walkontable/src/overlay/table/_base.js

```diff
@@ -0,0 +1,13 @@
+import Table from '../../table';
+
+class OverlayTable extends Table {
+  alignOverlaysWithTrimmingContainer() {
```

Comment:
There should probably a comment explaining why these empty methods are here.

Reply:
I will solve it differently. These methods are only used in master `Table`, so I will make a class for that and use them there.